.github/workflows/backend.yml

```diff
@@ -101,47 +101,10 @@ jobs:
           cat clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip.sha256 >> SHA256sum.txt
           echo " clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip" >> SHA256sum.txt
           sha256sum -c SHA256sum.txt
-          unzip clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip
-      - run: ./clj-kondo --version
+          unzip -d /usr/local/bin clj-kondo-${CLJ_KONDO_VERSION}-linux-static-amd64.zip
+      - run: clj-kondo --version
```

Comment:
`bin/kondo.sh` invokes `clj-kondo`, not `./clj-kondo`. So this unzips the `clj-kondo` binar to `/user/local/bin` so that it'll be in `$PATH`.